File: app/src/main/java/com/glsebastiany/bakingapp/repository/model/Ingredient.java
package com.glsebastiany.bakingapp.repository.model;


import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import org.parceler.Parcel;

@Parcel
public class Ingredient {

    @SerializedName("quantity")
    @Expose
    protected Float quantity = null;

    @SerializedName("measure")
    @Expose
    protected String measure = null;

    @SerializedName("ingredient")
    @Expose
    protected String ingredient = null;

    public Float getQuantity() {
        return quantity;
    }

    public void setQuantity(Float quantity) {
        this.quantity = quantity;
    }

    public String getMeasure() {
        return measure;
    }

    public void setMeasure(String measure) {
        this.measure = measure;
    }

    public String getIngredient() {
        return ingredient;
    }

    public void setIngredient(String ingredient) {
        this.ingredient = ingredient;
    }
}

File: app/src/main/java/com/glsebastiany/bakingapp/repository/model/Ingredient.kt
package com.glsebastiany.bakingapp.repository.model


import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import org.parceler.Parcel

@Parcel
class Ingredient {

    @SerializedName("quantity")
    @Expose
    var quantity: Float? = null

    @SerializedName("measure")
    @Expose
    var measure: String? = null

    @SerializedName("ingredient")
    @Expose
    var ingredient: String? = null

}





